package.json

```diff
@@ -13,7 +13,7 @@
     "flow": "lerna run flow",
     "codecov": "codecov",
     "publish:beta": "lerna publish --npm-tag beta --exact",
-    "publish:canary": "lerna publish --canary --exact patch --no-git-tag-version --no-push --npm-client npm",
+    "publish:canary": "lerna publish --canary --exact patch --no-git-tag-version --no-push",
```

Comment:
What's that?

Reply:
I agree, it should be done in another PR.

FYI: https://github.com/lerna/lerna/blob/master/commands/publish/CHANGELOG.md#372-2018-12-21

Reply:
If we keep those flags the script pops an error :

<img width="616" alt="Screenshot 2019-12-03 at 16 25 18" src="https://user-images.githubusercontent.com/15958334/70064276-8e77ca80-15e9-11ea-80da-83dc49a0729a.png">

Reply:
Introduced there: https://github.com/CoorpAcademy/components/pull/1638